neqo-qpack/src/encoder.rs

```diff
@@ -546,6 +546,41 @@ mod tests {
             // We will try to really change the table only when we send the change capacity instruction.
             self.encoder.send(&mut self.conn)
         }
+
+        pub fn insert(&mut self, header: &[u8], value: &[u8], inst: &[u8]) {
```

Comment:
Do you want to start using this more?  I see quite a few cases that might benefit from this.

Reply:
Yes. I am planning to use it more, but in a separate PR. This PR is already too big for such a small change.